docs/UserGuide.md

```diff
@@ -764,22 +972,34 @@ We will now teach you how to effectively use the `sort` command.
 
 </div>
 
+`sort` is one of the most complicated commands to get your head around. Perhaps a scenario can help you understand better!
+
+##### Scenario:
+
+Suppose you want to arrange contacts such from those that take the most number of common modules as you to the least. 
```

Comment:
Sentence does not make sense.

"Suppose you want to arrange your contacts based on the number of common modules they have with you, from those who have the most in common to those who have the least."